lamindb/_finish.py

```diff
@@ -19,7 +21,59 @@
 
 
 def get_save_notebook_message() -> str:
-    return f"Please save the notebook in your editor (shortcut `{get_shortcut()}`) within 2 sec before calling `finish()`"
+    return f"please save the notebook in your editor (shortcut `{get_shortcut()}`)"
+
+
+# this code was originally in nbproject by the same authors
+def check_consecutiveness(
```

Comment:
Maybe leave this for me to move later? Because it becomes confusing otherwise.

Reply:
I mean i will get rid of the `nbproject` dependency later.

Reply:
I'm doing it because I'm adding a new parameter and don't want to make a new nbproject release.

The function is only used here; so it won't be mistakingly copied over a second time.